app/javascript/packages/validated-field/validated-field-element.ts

```diff
@@ -117,20 +119,38 @@ class ValidatedFieldElement extends HTMLElement {
    */
   getOrCreateErrorMessageElement(): Element {
     if (!this.errorMessage) {
-      this.errorMessage = this.ownerDocument.createElement('div');
-      this.errorMessage.classList.add('usa-error-message');
-      if (this.descriptorId) {
-        this.errorMessage.id = this.descriptorId;
+      this.errorMessage =
+        (this.descriptorId && this.ownerDocument.getElementById(this.descriptorId)) ||
+        this.querySelector('.usa-error-message');
+
+      if (!this.errorMessage) {
+        this.errorMessage = this.ownerDocument.createElement('div');
+        this.errorMessage.classList.add('usa-error-message');
+        if (this.descriptorId) {
+          this.errorMessage.id = this.descriptorId;
+        }
+        if (this.input && TEXT_LIKE_INPUT_TYPES.has(this.input.type)) {
+          this.errorMessage.style.maxWidth = `${this.input.offsetWidth}px`;
+        }
+
+        this.inputWrapper?.appendChild(this.errorMessage);
       }
-      if (this.input && TEXT_LIKE_INPUT_TYPES.has(this.input.type)) {
-        this.errorMessage.style.maxWidth = `${this.input.offsetWidth}px`;
-      }
-
-      this.inputWrapper?.appendChild(this.errorMessage);
     }
 
     return this.errorMessage;
   }
+
+  /**
+   * Focus on this input if it's invalid and another error element
+   * does not have focus.
+   *
+   * @param isValid Whether input is valid.
+   */
+  private focusOnError(isValid: boolean) {
+    if (!(isValid && document.activeElement?.classList.contains('usa-input--error'))) {
```

Comment:
I think this `&&` should be an `||`, since currently it's not respecting that another element may already have focus.

```suggestion
    if (!isValid && !document.activeElement?.classList.contains('usa-input--error')) {
```

Reply:
Also, this is meant to be covered [by a test case in the ValidatedField element specs](https://github.com/18F/identity-idp/blob/8daf6bf15f91aed2f99685b269a5f54e2bee9175/app/javascript/packages/validated-field/validated-field-element.spec.ts#L85-L96), but it looks like there's a flaw in that code. Would you mind fixing it while you're here?

```diff
diff --git a/app/javascript/packages/validated-field/validated-field-element.spec.ts b/app/javascript/packages/validated-field/validated-field-element.spec.ts
index b7340ba9a..9937b2d3c 100644
--- a/app/javascript/packages/validated-field/validated-field-element.spec.ts
+++ b/app/javascript/packages/validated-field/validated-field-element.spec.ts
@@ -31,7 +31,7 @@ describe('ValidatedFieldElement', () => {
       </div>
     `;
 
-    const form = document.createElement('form');
+    const form = document.querySelector('form') || document.createElement('form');
     form.appendChild(element);
     document.body.appendChild(form);
 ```

Reply:
Done.